### PR TITLE
increased resolver test case

### PR DESCRIPTION
**Pull request by keiya01**

### CODE REVIEW

File: src/useFieldArray.test.tsx [hunk 1]
```diff
@@ -678,6 +678,54 @@ describe('useFieldArray', () => {
         ]),
       );
     });
+
+    describe('with resolver', () => {
+      it('should be called resolver when formState.isValid true', async () => {
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:683]: __
```suggestion
      it('should invoke resolver when formState.isValid true', async () => {
```

File: src/useFieldArray.test.tsx [hunk 2]
```diff
@@ -678,6 +678,54 @@ describe('useFieldArray', () => {
...
+        expect(resolver).toBeCalledWith(
+          {
+            test: [{ id: '0', value: '1' }],
+          },
+          undefined,
+          false,
+        );
+      });
+
+      it('should not be called resolver when formState.isValid false', () => {
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:710]:
```suggestion
      it('should not invoke resolver when formState.isValid false', () => {
```

File: src/useFieldArray.test.tsx [hunk 3]
```diff
@@ -1047,6 +1095,54 @@ describe('useFieldArray', () => {
       // @ts-ignore
       HTMLInputElement.prototype.removeEventListener.mockRestore();
     });
+
+    describe('with resolver', () => {
+      it('should be called resolver when formState.isValid true', async () => {
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:1100]:
```suggestion
      it('should invoke resolver when formState.isValid true', async () => {
```

File: src/useFieldArray.test.tsx [hunk 4]
```diff
@@ -1047,6 +1095,54 @@ describe('useFieldArray', () => {
...
+        expect(resolver).toBeCalledWith(
+          {
+            test: [{ id: '0', value: '1' }],
+          },
+          undefined,
+          false,
+        );
+      });
+
+      it('should not be called resolver when formState.isValid false', () => {
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:1127]:
```suggestion
      it('should not invoke resolver when formState.isValid false', () => {
```

File: src/useFieldArray.test.tsx [hunk 5]
```diff
@@ -1757,6 +1853,60 @@ describe('useFieldArray', () => {
         ]),
       );
     });
+
+    describe('with resolver', () => {
+      it('should be called resolver when formState.isValid true', async () => {
```
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:1858]:
```suggestion
      it('should invoke resolver when formState.isValid true', async () => {
```

File: src/useFieldArray.test.tsx (hunk 1 above)
Comment (bluebill1049) [inline src/useFieldArray.test.tsx:683]: minor: 
```
should invoke 
should not invoked
```

**Review by bluebill1049 — DISMISSED** [review]
LGTM! Thanks @keiya01

**Review by bluebill1049 — APPROVED** [review]
Thanks for the update.

**Comment by keiya01** [comment]
@bluebill1049 Thank you for your review.

**Pull request merged**